xml/libvirt_managing.xml

```diff
@@ -631,8 +631,10 @@
        made read-only, while a new qcow2 file is created to hold the changes.
        The original file is sometimes called a 'backing' or 'base' file, while
        the new file with all the changes is called an 'overlay' or 'derived'
-       file. External snapshots are useful when performing backups of
-       &vmguest;s. For more information on external snapshots in &qemu;, refer
+       file. External snapshots are useful when performing backups of &vmguest;s.
+       However, External snapshots could not be supported by &vmm;, and
```

Comment:
I would rephrase this sentence to: 'However, external snapshots are not supported by &vmm;, and cannot be deleted by <command>virsh</command> directly.' (there also was a small typo in 'directly')

Reply:
Changed as request.